pkgs/misc/vscode-extensions/default.nix

```
@@ -727,6 +727,18 @@ let
         };
       };
 
+      usernamehw.errorlens = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "errorlens";
+          publisher = "usernamehw";
+          version = "3.2.4";
+          sha256 = "0caxmf6v0s5kgp6cp3j1kk7slhspjv5kzhn4sq3miyl5jkrn95kx";
+        };
+        meta = {
+          license = lib.licenses.unfree;
```

```suggestion
          license = lib.licenses.mit;
```

See https://github.com/usernamehw/vscode-error-lens/blob/master/package.json#L7